litmus-portal/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -153,7 +155,16 @@ func CreateChaosWorkflow(input *model.ChaosWorkFlowInput, r store.StateData) (*m
 
 	workflow_id := uuid.New().String()
 
+	var workflow map[string]interface{}
+	err := json.Unmarshal([]byte(input.WorkflowManifest), &workflow_id)
+	if err != nil {
+		return nil, err
+	}
+
 	newWorkflowManifest, _ := sjson.Set(input.WorkflowManifest, "metadata.labels.workflow_id", workflow_id)
+	if strings.ToLower(workflow["kind"].(string)) == "cronworkflow" {
+		newWorkflowManifest, _ = sjson.Set(input.WorkflowManifest, "spec.workflowMetadata.labels.workflow_id", workflow_id)
```

Comment:
Is this correct `workflowMetadata` ?

Reply:
Yes, tested it. This is in the docs https://argoproj.github.io/argo/cron-workflows/